src/middleware.ts

```diff
@@ -19,7 +19,17 @@ import sharify from "sharify"
 import siteAssociation from "artsy-eigen-web-association"
 import timeout from "connect-timeout"
 import bodyParser from "body-parser"
-import config from "./config"
+import {
+  APP_TIMEOUT,
+  IP_DENYLIST,
+  NODE_ENV,
+  CLIENT_ID,
+  CLIENT_SECRET,
+  API_URL,
+  SEGMENT_WRITE_KEY_SERVER,
+  SENTRY_PRIVATE_DSN,
+} from "./config"
+import * as config from "./config"
```

Comment:
This is likely a micro optimization but perhaps move the second import up and then initialize vars from that single import?

Reply:
This is required so that we can pass the entire config to `@artsy/passport` as we were previously. In the future I would love to purposefully pass the values we need to `@artsy/passport`.

Reply:
https://github.com/artsy/force/pull/8062/files#diff-0626369278c89781505d2c04865ecce1302e979271bcfb471e029c19addc13a0R205